Gopkg.lock

```diff
@@ -191,6 +191,12 @@
   revision = "2dd44038f0b95ae693b266c5f87593b5d2fdd78d"
   version = "v2.5.0"
 
+[[projects]]
+  name = "github.com/fsnotify/fsnotify"
```

Comment:
@cjwagner we should use this in the config agent to drive down config reload times :-)

Reply:
I've been meaning to do this, and now somebody has vendored this for me :joy:

Reply:
I'm always worried that it's going to miss some event somehow. With fs watchers there are always edge cases. Polling erases the worry :)

Reply:
sure, but we can poll *with* notifying. we would *actually* miss some events only notifying. we've had problems with this with some kubernetes tests :-)

Reply:
Oh yeah, polling + notifying is fine with me. Not sure how important it is to get <1 min config reload times tho.

Reply:
frequently we update prow config to patch some misconfigured test and then someone sees the config update message on github and triggers a new test but the config hasn't actually been reloaded yet. fast reload times for job config would be nice, but definitely not important

Reply:
I should add a poll to this loop ...